internal/provider/data_source_cloudexport_item_test.go

```diff
@@ -1,33 +1,35 @@
 package provider_test
 
 import (
+	"fmt"
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 )
 
-// Note: values checked in tests below are provided by fake API Server from test-data.json
-// (running in background).
-
-//nolint: dupl
 func TestDataSourceCloudExportItemAWS(t *testing.T) {
 	t.Parallel()
+
+	server := newTestAPIServer(t, makeInitialCloudExports())
+	server.Start()
+	defer server.Stop()
+
 	resource.UnitTest(t, resource.TestCase{
-		PreCheck:          func() { checkAPIServerConnection(t) },
 		ProviderFactories: providerFactories(),
 		Steps: []resource.TestStep{
 			{
-				Config: testCloudExportDataSourceItems,
+				Config: makeTestCloudExportDataSourceItems(server.URL()),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(ceAWSDS, "id", "1"),
 					resource.TestCheckResourceAttr(ceAWSDS, "type", "CLOUD_EXPORT_TYPE_KENTIK_MANAGED"),
 					resource.TestCheckResourceAttr(ceAWSDS, "enabled", "true"),
 					resource.TestCheckResourceAttr(ceAWSDS, "name", "test_terraform_aws_export"),
 					resource.TestCheckResourceAttr(ceAWSDS, "description", "terraform aws cloud export"),
-					resource.TestCheckResourceAttr(ceAWSDS, "api_root", "http://localhost:8080/api"),
-					resource.TestCheckResourceAttr(ceAWSDS, "flow_dest", "http://localhost:8080/flow"),
 					resource.TestCheckResourceAttr(ceAWSDS, "plan_id", "11467"),
 					resource.TestCheckResourceAttr(ceAWSDS, "cloud_provider", "aws"),
+					resource.TestCheckResourceAttr(ceAWSDS, "bgp.0.apply_bgp", "true"),
+					resource.TestCheckResourceAttr(ceAWSDS, "bgp.0.use_bgp_device_id", "dummy-device-id"),
+					resource.TestCheckResourceAttr(ceAWSDS, "bgp.0.device_bgp_type", "dummy-device-bgp-type"),
```

Comment:
Is the indentation on lines 36 - 44 really off, or is it just a GitHub UI artifact?

(Note: the comment must be here, because stupid does not allow to put it on line 36)

Reply:
It is GH UI artifact. CI requires the code to be gofmted